bedrock/firefox/templates/firefox/family/includes/modules/bullying.html

```diff
@@ -33,8 +33,7 @@ <h4 class="c-title-uppercase">What to do:</h4>
 
       <aside class="c-blurb">
         <h4 class="c-blurb-title">Fun fact:</h4>
-        <p>Trolls originated in Scandinavian folklore. Most internet trolls originate
-          in their mom’s basement.</p>
+        <p>A 2019 study indicated that nationwide, about 16 percent of students in grades 9–12 have experienced cyberbullying. <span class="c-blurb-source">Source: stopbullying.gov</span></p>
```

Comment:
Nit: include the country where this study was conducted?

```suggestion
        <p>A 2019 US study indicated that nationwide, about 16 percent of students in grades 9–12 have experienced cyberbullying. <span class="c-blurb-source">Source: stopbullying.gov</span></p>
```